the-blue-alliance-ios/View Elements/Awards/AwardTableViewCell.swift

```diff
@@ -9,25 +9,40 @@ class AwardTableViewCell: UITableViewCell, Reusable {
         }
     }
     var teamKeySelected: ((_ teamKey: String) -> Void)?
+    private let awardNameLabel = UILabel()
+    private let awardsFlexView = FlexLayoutView()
+    
+    // Mark: - Init
 
-    // MARK: - Reusable
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
 
-    static var nib: UINib? {
-        return UINib(nibName: String(describing: self), bundle: nil)
-    }
+        awardNameLabel.font = .systemFont(ofSize: 16, weight: .medium)
+        awardNameLabel.numberOfLines = 0
+        awardNameLabel.translatesAutoresizingMaskIntoConstraints = false
+        self.addSubview(awardNameLabel)
+        awardNameLabel.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 20).isActive = true
+        awardNameLabel.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -20).isActive = true
+        awardNameLabel.topAnchor.constraint(equalTo: topAnchor, constant: 20).isActive = true
+        awardNameLabel.setContentHuggingPriority(UILayoutPriority(998), for: .vertical)
 
-    // MARK: - Interface Builder
+        awardsFlexView.horizontalSpacing = 10
```

Comment:
Can we do this kind of setup in `lazy var`s? That's usually the pattern for things like this. And then in the init just add things to the view and setup AL constraints and whatnot.